lms/static/scripts/frontend_apps/api-types.ts

```diff
@@ -147,8 +147,13 @@ export type YouTubeVideoInfo = {
  * Dashboard-related API types
  */
 
+/**
+ * Represents dates exposed by the backend as string, in ISO format
```

Comment:
```suggestion
 * Represents dates exposed by the backend as string, in ISO 8601 format
```

Just in case a reader is not certain what "ISO format" means, this addition makes it easy to look it up.